ansible_image_validation/validation-playbooks/per-vm-validation.yaml

```diff
@@ -141,6 +141,82 @@
       is_rhui_package_present : true
     when: rhui_package_count != "0"
 
+##Repo Validation Task
+
+- name: Check if offertype is equal to byol
+  lineinfile:
+    path: "{{err_folder}}/err_msgs.log"
+    line: "\n  Client Package is present in byol Offertype Image"
+    create: yes
+    state: present 
+  when: offer_type =='byol' and rhui_package_count != "0"
+
+- name: Check Condition offer_type == byol
+  ansible.builtin.debug:
+    var: offer_type == 'byol'  
+
+- name: Check the value of client package
+  shell: rpm -qa | grep rhui
+  register: output_client_package
+  when: offer_type!='byol'
+
+- debug:
+    var: output_client_package
+
+- name: Check if the client package is present according to the offertype when offertype is not equal to base or byol
+  shell: |
```

Comment:
shouldn't the validation instead check that 'sap-ha' offertype has 'sapha' package? 

If there was 'eus' package installed for 'sapha', the validation would pass, isnt' it?

Reply:
Validation is added for all the offertypes